__tests__/utils.tsx

```diff
@@ -93,12 +95,8 @@ describe('getPathInfo()', () => {
   })
 
   test('returns typename and currentPath from api.serlo.org', async () => {
-    mockFetch({
-      [apiEndpoint]: createApiResponse({
-        __typename: 'Article',
-        alias: '/current-path',
-      }),
-    })
+    serverMock('Article', returnResponseText('/current-path'))
+    
```

Comment:
```suggestion
    serverMock(apiEndpoint, returnApiResponse({
        __typename: 'Article',
        alias: '/current-path',
      }))
```

Du musst noch folgendes in dieser Datei definieren (oder in _helper.ts):

```
function returnResponseApi(data: unknown) {
  return returnResponseJson({ data })
}
```